.all-contributorsrc

```diff
@@ -159,6 +159,8 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/10661605?v=4",
       "profile": "https://aka.ms/helderpinto",
       "contributions": [
+        "code",
```

Comment:
@flanakin And also this one

Reply:
@flanakin, I didn't use any local NPM command. I didn't find any docs describing how these entries are expected to be updated. That's why I added them manually.